packages/wifi_basic/lib/wifi_info.dart

```diff
@@ -0,0 +1,62 @@
+part of 'wifi_basic.dart';
+
+class WiFiInfo {
+  static const Equality _equality = DeepCollectionEquality();
+
+  final String ssid;
+  final String bssid;
+  final WiFiNetworkSecurity security;
+  final bool isHidden;
+  final int rssi;
+  final double signalStrength;
+  final bool hasInternet;
+  final WiFiGenerations generation;
+
+  bool get isNull => ssid.isEmpty;
+
+  @visibleForTesting
+  WiFiInfo.fromMap(Map map)
```

Comment:
can we not keep it private?

Reply:
and if it is public, do we need `visibleForTesting` annotation?

Reply:
also can we make it const?

Reply:
The `visibleForTesting` requires the thing in question to be "public". But it's only public inside test classes. For the rest of the code, the property is considered private.

And yes, it probably can be `const`.